processing/data_collection/gazette/spiders/pr_cascavel.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from dateparser import parse
+import datetime as dt
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCascavelSpider(BaseGazetteSpider):
+    TERRITORY_ID = '4104808'
+    name = 'pr_cascavel'
+    allowed_domains = ['cascavel.pr.gov.br']
+    start_urls = ['http://www.cascavel.pr.gov.br/servicos/orgao_oficial.php']
+    download_url = 'http://www.cascavel.pr.gov.br/anexos/{}'
+
+    def parse(self, response):
+        """
+        @url http://www.cascavel.pr.gov.br/servicos/orgao_oficial.php
+        @returns items 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+        for row in response.css('table tr')[1:]:
+            cols = row.css('td')
+            edition = cols[0].css('font::text').extract()
```

Comment:
You could use xpath instead of css. I think it is clearer and easier to maintain.
```
edition = row.xpath('.//td[1]//font//text()').extract_first()
date = row.xpath('.//td[2]//font//text()').extract_first()
urls = row.xpath('.//td[3]//a')
```

Also, I noticed that you never use `edition` variable. It could be removed.